### PR TITLE
edits and modifications to wording/grammar for initialization chap (5)

### PR DESCRIPTION
**Pull request by kkeene44**

### CODE REVIEW

**Comment by kkeene44** [comment]
@davegill @dudhia 
Please review the changes I've made to this chapter when you get a chance. Most of the mods are just wording/grammar.

File: initialization.tex [hunk 1]
```diff
@@ -118,68 +118,67 @@ \section{Initialization for Idealized Conditions}
...
-equations for $p$ and $\rho$ in the vertical integration, and they are
-solved by iteration.  The dry pressure on input sounding levels is
+equations for $p$ and $\rho$ in the vertical integration, solved by iteration.  
+Dry pressure on input sounding levels is
 computed by integrating the hydrostatic relation down from the top,
 excluding the vapor component.
 
-Having computed the full pressure (dry plus vapor) and dry air pressure
-on the input sounding levels, the pressure at the model top ($p_{t}$)
+While the full pressure (dry, plus vapor) and dry air pressure is
```
Comment (davegill) [inline initialization.tex]: @kkeene44 
maybe this one is "are" computed?

File: initialization.tex [hunk 2]
```diff
@@ -118,68 +118,67 @@ \section{Initialization for Idealized Conditions}
...
-the inverse density $\alpha_d$ is diagnosed from the geopotential using
-this dry hydrostatic relation, and the pressure is diagnosed from the equation
+inverse density $\alpha_d$ is diagnosed from the geopotential using
+this dry hydrostatic relation, and pressure is diagnosed from the equation
 of state using the inverse density $\alpha_d$ and the prognostic potential
 temperature $\theta_m$.  Thus, the ARW's prognostic variables $\mu_d$,
-$\theta_m$, and $\phi$ are in exact hydrostatic balance for the model
-equations (to machine roundoff).
+$\theta_m$, and $\phi$ are in exact hydrostatic balance (to machine roundoff) 
+for the model equations.
```
Comment (davegill) [inline initialization.tex]: @kkeene44 
"with" the model equations

File: initialization.tex [hunk 3]
```diff
@@ -306,14 +306,14 @@ \subsection{Vertical Interpolation and Extrapolation}
...
+however, near the model surface, it is possible to have an inconsistency between the input
+surface pressure (which is largely based on the input surface elevation) and the ARW surface
+pressure (which could have a significantly higher-resolution topography).  These inconsistencies
 may lead to an extrapolation.  The default behavior for extrapolating the horizontal winds and
-the relative humidity below the known surface is to keep the values constant, with zero vertical gradient.
-For the potential temperature, by default a -6.5 $K/km$ lapse rate for the temperature is applied.
-The vertical interpolation of the geopotential field is optional and is
+the relative humidity below the known surface keeps the values constant, with zero vertical gradient.
+For potential temperature, a default of -6.5 $K/km$ lapse rate is applied.
+Vertical interpolation of the geopotential field is optional and is
```
Comment (davegill) [inline initialization.tex]: PLACEHOLDER SO I KNOW WHERE I AM

File: initialization.tex [hunk 4]
```diff
@@ -410,71 +410,70 @@ \subsection{Generating Lateral Boundary Data}
...
 place-holders for the nested boundary data for the fine grids.
 The width of the lateral
-boundary along each of the four sides is user selectable at run-time.
+boundary along each of the four sides is user-selectable at run-time.
 
 \subsection{Masking of Surface Fields}
 
-Some of the meteorological and static fields are ``masked''.  A masked field is one in which
-the values are typically defined only over water (e.g., sea surface temperature) or defined
+Some of the meteorological and static fields are ``masked."  A masked field is one in which
```
Comment (davegill) [inline initialization.tex]: The period should be outside of the quotes

File: initialization.tex [hunk 5]
```diff
@@ -410,71 +410,70 @@ \subsection{Generating Lateral Boundary Data}
...
-soil data.   While it is typical to try to match the ARW soil scheme with
+by an external model.  WPS does not vertically interpolate
+soil data.  While it is typical to attempt to match the ARW soil scheme with
 the incoming data, that is not a requirement.  Pre-processor {\it real} will vertically interpolate 
 (linear in depth below the ground) from the incoming levels to the requested soil layers to be
 used within the model.
 
 \section{Digital Filtering Initialization}
 
 Version 3 of the ARW provides a digital filtering initialization (DFI) to
```
Comment (davegill) [inline initialization.tex]: We should swap "Version 3 of the ARW" with '"ARW".

**Comment by davegill** [comment]
@kkeene44 @mgduda 
Kelly,

Following Michael's use of "GRIB-formatted data" in the User's Guide, I added that in a couple of places (as opposed to GRIB data).
With your suggestions, apparently my favorite band has always been "The The".
I also appreciate how you transmogrified some unnecessarily weird phrases.

I am good with these mods to CH 5.

File: initialization.tex [hunk 6]
```diff
@@ -336,9 +336,9 @@ \subsection{Perturbation State}
 \end{equation}
 
 \noindent Starting with the reference state fields 
-(\ref{init_pbar}, \ref{init_alphabar}, and \eqref{init_mubar}) and the perturbation (\ref{init_muprime}),
+(\ref{init_pbar}, \ref{init_alphabar}, and \ref{init_mubar}) and the perturbation (\ref{init_muprime}),
```
Comment (kkeene44) [inline initialization.tex]: @davegill 
Did you change this? If I did, I didn't mean to. Just want to make sure it's correct.

File: initialization.tex [hunk 7]
```diff
@@ -410,71 +410,70 @@ \subsection{Generating Lateral Boundary Data}
...
 place-holders for the nested boundary data for the fine grids.
 The width of the lateral
-boundary along each of the four sides is user selectable at run-time.
+boundary along each of the four sides is user-selectable at run-time.
 
 \subsection{Masking of Surface Fields}
 
-Some of the meteorological and static fields are ``masked''.  A masked field is one in which
-the values are typically defined only over water (e.g., sea surface temperature) or defined
+Some of the meteorological and static fields are "masked".  A masked field is one in which
```
Comment (kkeene44) [inline initialization.tex]: @davegill 
In American English, punctuation should always go inside the quotes.
https://www.grammarbook.com/punctuation/quotes.asp (see rule 4).
So this should be "masked."

File: initialization.tex (hunk 6 above)
Comment (davegill) [reply initialization.tex]: I did not change it either. Let's see what the pdf looks like afterwards.

**Pull request merged**